interface/models.py

```diff
@@ -72,13 +72,16 @@ class Submission(models.Model):
                                    null=True)
     user = models.ForeignKey(User, on_delete=models.PROTECT, null=True)
     output = models.CharField(max_length=4096, default='none')
+    review_message = models.CharField(max_length=1024, default='none')
```

Comment:
Let's be more generous for the talkative TA, it costs us nothing to give them more space :)